### ISSUE TITLE
Get("~/.well-known/jwks") results in a 404

### ISSUE THREAD

**Issue by DominusFlevit** [FastEndpoints/FastEndpoints#763]
I suspect that the ~ character is messing up the route.

```
 internal sealed class JWKSEndpoint : EndpointWithoutRequest<JWKSResponse>
 {
     private readonly SecretClient _secretClient;
     public JWKSEndpoint(SecretClient secretClient)
     {
         _secretClient = secretClient;
     }
     public override void Configure()
     {
         Options(x => x.CacheOutput(p => p.Expire(TimeSpan.FromHours(4))));
         Get("~/.well-known/jwks");
         AllowAnonymous();
     }
     public override async Task HandleAsync(CancellationToken ct)
     {
         await SendAsync(new JWKSResponse(), cancellation: ct);
     }
 }
```

**Comment by dj-nitehawk** [comment]
just tried the following:
```cs
var bld = WebApplication.CreateBuilder(args);
bld.Services
   .AddFastEndpoints()
   .SwaggerDocument();

var app = bld.Build();
app.UseFastEndpoints()
   .UseSwaggerGen();
app.Run();

sealed class MyEndpoint : EndpointWithoutRequest
{
    public override void Configure()
    {
        Get("~/.well-known/jwks");
        AllowAnonymous();
    }

    public override async Task HandleAsync(CancellationToken c)
    {
        await SendAsync("hello world");
    }
}
```

and i get the correct response:

![screen-record-1726676736](https://github.com/user-attachments/assets/fdfc7dfa-daf7-4426-b9b0-bb211d5ac012)

**Comment by DominusFlevit** [comment]
Sorry for not being more explicit. The 404 happens when you make the call via Execute in the SwaggerUI:
<img width="753" alt="image" src="https://github.com/user-attachments/assets/4f910a5e-1d7e-4d6e-a066-23aa1b073271">

**Comment by dj-nitehawk** [comment]
fixed in `v5.29.0.5-beta`
thanks for reporting 🙏
